### ISSUE TITLE
UUID or ID on attack-datasouurces

### ISSUE THREAD

**Issue by adulau** [mitre-attack/attack-datasources#62]
Thanks for the project. It's a very good idea. 

- Will you add an fixed/permanent UUID or ID in the sources? 

It could be useful for many project to reuse the same data source description or create relationships on a permanent basis (just like we do in [CyCAT.org](https://www.cycat.org/)).

**Comment by adampennin** [comment]
Yup! Those fields weren't in my ATT&CK EU slides this morning, but both should be a part of making data sources an object (somewhat similar to Mitigations).
